hawkauthlib/__init__.py

```diff
@@ -178,16 +174,26 @@ def verify_payload(request, params=None, algorithm=None):
     if params is None:
         params = utils.parse_authz_header(request, {})
 
-    return utils.strings_differ(params["hash"], hash_payload(request, params, algorithm))
+    return not utils.strings_differ(params["hash"], hash_payload(request, params, algorithm))
```

Comment:
This should return `True` if the strings to *not* differ, right?

Reply:
according to the naming, a double negatire, yeah. Interesting that tests pass before but fail now.